### PR TITLE
Wizard ask password dialog

### PR DESCRIPTION
**Pull request by notmike-5**
Moved to its own branch.
See #2236 
#2208

### CODE REVIEW

File: wizard/WizardModeSelection.qml [hunk 1]
```diff
@@ -57,16 +57,16 @@ Rectangle {
 
             WizardHeader {
                 title: qsTr("Mode selection.") + translationManager.emptyString
-                subtitle: qsTr("Please select the statement that best matches you.") + translationManager.emptyString
+                subtitle: qsTr("Please select the statement that best matches your desired setup.") + translationManager.emptyString
```
Comment (erciccione) [inline wizard/WizardModeSelection.qml]: Is this change really needed? I'm fine with editing the text if necessary, but keep in mind that everytime a string is changed even slightly, the matching string in all the language files will be resetted and will have to be retranslated. This for *more than 30 languages*.

File: wizard/WizardModeSelection.qml [hunk 2]
```diff
@@ -121,8 +121,8 @@ Rectangle {
             }
 
             WizardMenuItem {
-                headerText: qsTr("Advanced mode") + translationManager.emptyString
-                bodyText: qsTr("Includes extra features like mining and message verification. The blockchain is downloaded to your computer.") + translationManager.emptyString
+                headerText: qsTr("Advanced mode (maximum privacy)") + translationManager.emptyString
+                bodyText: qsTr("Includes extra features like mining and message verification. By default, the blockchain is downloaded to your computer.") + translationManager.emptyString
```
Comment (erciccione) [inline wizard/WizardModeSelection.qml]: See my comment above

File: wizard/WizardModeSelection.qml (hunk 1 above)
Comment (notmike-5) [reply wizard/WizardModeSelection.qml]: I don't really have a preference. This came from an issue #2208. I didn't create a branch initially and now my downstream master has this modded file in it. I'm happy to discard it, close the issues, and reclone to the latest upstream master, but didn't want to do that before resolving.

File: wizard/WizardModeSelection.qml (hunk 2 above)
Comment (notmike-5) [reply wizard/WizardModeSelection.qml]: If this is closed, it is related to #2236 which could also be closed.
